components/admin_console/admin_definition.jsx

```diff
@@ -2961,7 +2978,10 @@ const AdminDefinition = {
                         label: t('admin.ldap.enableAdminFilterTitle'),
                         label_default: 'Enable Admin Filter:',
                         isDisabled: it.any(
+
+                            // Should this one be removed?
                             it.not(it.isSystemAdmin),
+                            it.not(it.userHasWritePermissionOnResource(RESOURCE_KEYS.AUTHENTICATION.LDAP)),
```

Comment:
No, only a "SystemAdmin" can update permissions which changes "SystemAdmins".  Therefore this check should remain. No additional check is necessary.